### ISSUE TITLE
Use a logger

### ISSUE THREAD

**Issue by Ryuno-Ki** [solid/node-solid-server#920]
Think about using a logger library. My favourite is [loglevel](https://www.npmjs.com/package/loglevel) (because it's simple and supports plugins).

[winston](https://www.npmjs.com/package/winston) is popular also.

This way you can fine-tune the logging.

**Comment by kjetilk** [comment]
yeah, it is on my whiteboard for v.next (which is when we seriously introduce new things). I really miss multi-level logging, so it will definitely go in there. But it seems like way too much work to add different loglevels in the current code, so I hope you will excuse me for removing the triage label :-)

**Comment by Ryuno-Ki** [comment]
You could start small like I did in https://github.com/matrix-org/matrix-js-sdk/pull/763 the other day.

**Comment by kjetilk** [comment]
So, it is not very likely we'd prioritize that highly in this code, but it is really needed in https://github.com/solid/acl-check/issues/6 so I'll have a look at those hopefully. If there are any others I should be aware of, please let me know over in that bug :-)
